node/src/fork_off.rs

```diff
@@ -0,0 +1,124 @@
+use std::io::Write;
+
+use hex_literal::hex;
+use remote_externalities::{Builder, Mode, OfflineConfig, OnlineConfig, SnapshotConfig, Transport};
+use sc_cli::CliConfiguration;
+use sc_service::Configuration;
+use sp_core::bytes::to_hex;
+
+const SKIPPED_PALLETS: [&str; 7] = [
+    "System",
+    "Session",
+    "Babe",
+    "Grandpa",
+    "GrandpaFinality",
+    "Authorship",
+    "Sudo",
+];
+
+const INCLUDED_PREFIXES: [[u8; 32]; 1] = [hex_literal::hex!(
+    "26aa394eea5630e07c48ae0c9558cef7b99d880ec681799c0cf30e8886371da9"
```

Comment:
I think it would be better, if we document included prefixes, since it's hard to tell what do they hold.
```suggestion
// Keys with this prefix will be preserved to save the account data    "26aa394eea5630e07c48ae0c9558cef7b99d880ec681799c0cf30e8886371da9"
```